npu.user.js

```diff
@@ -606,8 +606,12 @@
         }
         var keepAlive = function() {
           window.setTimeout(function() {
+              var pages = ["main.aspx", "main.aspx?ismenuclick=true&ctrl=inbox", "main.aspx?ctrl=0203&ismenuclick=true", "main.aspx?ismenuclick=true&ctrl=outbox"];
+              var index = Math.floor(Math.random())%pages.length;
```

Comment:
`Math.floor(Math.random())` returns zero every time so `index` will always be zero.

Reply:
Thanks, I totally overlooked that and fixed it.